pyproject.toml

```diff
@@ -9,7 +9,7 @@ style = "semver"
 pattern = "default-unprefixed"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
```

Comment:
Are we sure about this? Our client still supports 3.7. Has this been consulted with anyone?

Reply:
Bumped as I was using the walrus operator. We can move back to the traditional assignment and conditional and keep the python dependency at 3.7, but should we make compromises to accommodate an unsupported version of Python?

Let me consult this in internal channels

Reply:
Don't get me wrong - I'm not saying that we shouldn't. But I think we need to gather more opinions on that

Reply:
Yup, I totally understand.
Have posted this in the API discussion channel